File: Leetcode/344. Reverse String/ReverseString.cpp
//풀이 1

vector<char> st(s.size());
int j = 0;

//for(int i = s.size()-1, j = 0; i>=0,j<s.size();i--,j++)
for(int i = s.size()-1; i>=0;i--){
    st[j] =s[i];
    j++;
}

for(int i = 0; i<st.size();i++){
    s[i]=st[i];
}

//풀이 2
reverse(s.begin(),s.end());

File: Leetcode/344. Reverse String/ReverseString.py
//풀이 1
first, last = 0, len(s)-1  #Initialize first and last

while first < last:
    s[first],s[last] = s[last],s[first]  #Swap first and last contents
    first += 1
    last -= 1
    
    
//풀이 2
s.reverse()
